### PR TITLE
Fix training NR pipelines

### PR DESCRIPTION
**Pull request by FlorentinD**

### CODE REVIEW

**Comment by FlorentinD** [comment]
[Fix using NodeRegression pipelines in the client](https://trello.com/c/SHZOWHUn/6882-fix-using-noderegression-pipelines-in-the-client)

File: changelog.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 ## Bug fixes
 
+* Fix training Node regession pipelines created via `gds.alpha.pipeline.nodeRegression.create`.
```
Comment (breakanalysis) [inline changelog.md:12]: I dont understand what bug this is fixing. Looks more like the whole feature was not exposed ?

Comment (breakanalysis) [inline changelog.md:12]:
```suggestion
* Fix training Node regression pipelines created via `gds.alpha.pipeline.nodeRegression.create`.
```

**Review by breakanalysis — CHANGES_REQUESTED** [review]
Changes look good, however I think the changelog should specify what its fixing.

File: changelog.md (hunk 1 above)
Comment (FlorentinD) [reply changelog.md:12]: It is a bug, as the feature was documented and therefore exposed

**Comment by FlorentinD** [comment]
Thank you for the review I extended the change log entry, but kept it as a bug-fix as the feature was exposed to the user via Documentation as well as the python classes were present

**Pull request merged**